scm-webapp/src/main/java/sonia/scm/api/v2/resources/ScmConfigurationToConfigDtoMapper.java

```diff
@@ -44,6 +47,14 @@
   @Inject
   private ResourceLinks resourceLinks;
 
+  @Mapping(target = "anonymousAccessEnabled", source = "anonymousMode", qualifiedByName = "mapAnonymousAccess")
+  public abstract ConfigDto map(ScmConfiguration scmConfiguration);
+
+  @Named("mapAnonymousAccess")
+  boolean mapAnonymousAccess(AnonymousMode anonymousMode) {
+    return anonymousMode != AnonymousMode.OFF;
+  }
```

Comment:
How does this work?

Reply:
I don't get the question. The configDto still has the field "anonymousAccessEnabled". When mapping this field the mapper uses `mapAnonymousAccess`. There are unit tests for this.